ecs-cli/modules/cli/compose/context/context.go

```diff
@@ -40,7 +40,8 @@ type Context struct {
 	EC2Client ec2client.EC2Client
 
 	// IsService would decide if the resource created by this compose project would be ECS Tasks directly or through ECS Services
-	IsService bool
+	IsService   bool
+	TaskRoleArn string
```

Comment:
Good question! Eventually, we will need either a wrapper around the libcompose context or another ECS task definition context to support ECS task definition fields that are not supported by libcompose.

For now, we can directly add `taskRokeArn := context.CLIContext.String(command.TaskRoleArnFlag)` in project.go transformTaskDefinition function.